### PR TITLE
exclude health checks from ga

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn review please. Same change done in [unicore-cms](https://github.com/universalcore/unicore-cms/pull/110)

File: springboard/utils.py [hunk 1]
```diff
@@ -8,6 +8,16 @@
 from elasticgit.search import RepoHelper
 
 
+default_excluded_paths = ['/health/', '/api/notify/']
+
+
+def excluded_path(path, excluded_paths):
+    excl_paths = excluded_paths.split(',') + default_excluded_paths
```
Comment (smn) [inline springboard/utils.py]: Don't like [`utils.config_list()`](https://github.com/universalcore/springboard/blob/develop/springboard/utils.py#L67)?

File: springboard/utils.py [hunk 2]
```diff
@@ -8,6 +8,16 @@
 from elasticgit.search import RepoHelper
 
 
+default_excluded_paths = ['/health/', '/api/notify/']
+
+
+def excluded_path(path, excluded_paths):
```
Comment (smn) [inline springboard/utils.py]: Tiny nitpick on this being named `is_excluded_path()` since it's a test that returns a boolean.

**Comment by smn** [comment]
:+1: happy for you to ignore the nitpicks.

**Pull request merged**